File: client/Client/Network/Messages/Session/DenySelectSectionMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DenySelectSectionMessage
    {
        public const string TYPE = "DenySelectSectionMessage";
        private string _reason;

        public string reason
        {
            get => _reason;
            set => _reason = value;
        }

        public DenySelectSectionMessage()
        {
        }

        public DenySelectSectionMessage(string reason)
        {
            this.reason = reason;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/DenySelectSectionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class DenySelectSectionMessage {
    public static final String TYPE = "DenySelectSectionMessage";

    private String reason;

    public DenySelectSectionMessage() {
    }

    public DenySelectSectionMessage(String reason) {
        this.reason = reason;
    }

    public String getReason() {
        return this.reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }
}
